tests/components/axis/test_switch.py

```diff
@@ -67,26 +75,28 @@ async def test_switches_with_port_cgi(hass):
 
     assert len(hass.states.async_entity_ids(SWITCH_DOMAIN)) == 2
 
-    relay_0 = hass.states.get(f"switch.{NAME}_doorbell")
-    assert relay_0.state == "off"
-    assert relay_0.name == f"{NAME} Doorbell"
-
-    relay_1 = hass.states.get(f"switch.{NAME}_relay_1")
-    assert relay_1.state == "on"
+    relay_1 = hass.states.get(f"{SWITCH_DOMAIN}.{NAME}_relay_1")
+    assert relay_1.state == STATE_ON
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)